assembly/std/index.ts

```diff
@@ -75,63 +140,94 @@ export function localExecution(
+ * @remarks
+ * Bytecode is a low-level representation of a smart contract's code that can be executed by the blockchain.
  *
- * @param address -
+ * @param address - The address of the contract's bytecode to retrieve.
+ *
+ * @returns The bytecode of the contract, serialized as a 'StaticArray<u8>'.
+ *
+ * @throws
+ * - if the given address is not a valid smart-contract address.
  *
- * @returns bytecode
  */
 export function getBytecodeOf(address: Address): StaticArray<u8> {
   return env.getBytecodeOf(address.toString());
 }
 
 /**
- * Check if the SC caller has a write access on it
+ * Determine if the caller has write access to the data stored in the called smart-contract.
+ *
+ * @remarks
+ * This function returns true exclusively when a new smart contract is deployed using the
+ * 'create_new_sc_address()' function.
+ * When calling {@link createSC}, the User or smart contract will be granted write
+ * access to the created SC, but this privilege is limited to the context of this specific operation.
+ *
+ * @returns Returns true if the caller has write access; false otherwise.
  *
  */
 export function callerHasWriteAccess(): bool {
   return env.callerHasWriteAccess();
 }
 
 /**
- * Checks if `function` exists in the bytecode stored at `address`
- * @param address -
- * @param func -
+ * Creates a new smart contract on the ledger using its bytecode representation.
+ *
+ * @remarks
+ * After executing this function, you will have write access on the newly generated contract.
+ *
+ * @see {@link callerHasWriteAccess} for more information.
+ *
+ * @param bytecode - The byte code of the contract to create.
+ *
+ * @returns The address of the newly created smart contract on the ledger.
+ *
  */
-export function functionExists(address: Address, func: string): bool {
-  return env.functionExists(address.toString(), func);
+export function createSC(bytecode: StaticArray<u8>): Address {
+  return new Address(env.createSC(bytecode));
 }
 
 /**
- * Creates a new smart contract.
+ * Checks if the given function exists in a smart-contract at the given address.
  *
- * Take a base64 string representing the module binary and create an entry in
- * the ledger.
+ * @param address - The address of the contract to search in.
+ * @param func - The name of the function to search.
  *
- * The context allow you to write in this smart contract while you're executing
- * the current bytecode.
+ * @returns true if the function exists, false otherwise.
  *
- * @param bytecode -
+ * @throws
+ * - if the given address is not a valid smart-contract address.
  *
- * @returns Smart contract address
  */
-export function createSC(bytecode: StaticArray<u8>): Address {
-  return new Address(env.createSC(bytecode));
+export function functionExists(address: Address, func: string): bool {
+  return env.functionExists(address.toString(), func);
 }
 
 /**
- * Generates an event
+ * Generates a string event that is then emitted by the blockchain and can be listened off-chain.
```

Comment:
I thought about something simpler
```suggestion
 * Generate a log in the Massa blockchain
```